Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3
+
+WORKDIR /usr/src/app
+COPY ./. .
+RUN pip install pipenv && pipenv install --system --ignore-pipfile
+EXPOSE 8000
+
+CMD [ "python", "src/server.py" ]
+
```

Comment:
Delete this whitespace, you have two whitespace lines (9 and 10), you only need one at the EOF (**E**nd **O**f **F**ile)

Reply:
done,
the PR is ready to be merged